src/lib.rs

```diff
@@ -92,71 +137,169 @@ impl<Page: fmt::Debug, Ret> PagerRet<Page, Ret> {
+
+    /// Creates a new [`TransitionStyle`] for horizontal page transitions.
+    ///
+    /// Uses default settings based on the provided UI's [style](egui::Ui::style),
+    /// with horizontal movement as the transition type.
     ///
-    /// It uses default settings mostly based on the provided ui's [style](egui::Ui::style),
-    /// but values of some fields (eg. [easing](TransitionStyle::easing)) are opinionated and may change slightly
-    /// between versions.
+    /// # Parameters
+    /// - `ui`: The current [`Ui`] context, used to derive default style settings.
     pub fn horizontal(ui: &Ui) -> Self {
         Self::new_with_type(ui, TransitionType::HorizontalMove)
     }
-    /// Create a new [`TransitionStyle`] animated by shifting vertically.
+
+    /// Creates a new [`TransitionStyle`] for vertical page transitions.
+    ///
+    /// Uses default settings based on the provided UI's [style](egui::Ui::style),
+    /// with vertical movement as the transition type.
     ///
-    /// It uses default settings mostly based on the provided ui's [style](egui::Ui::style),
-    /// but values of some fields (eg. [easing](TransitionStyle::easing)) are opinionated and may change slightly
-    /// between versions.
+    /// # Parameters
+    /// - `ui`: The current [`Ui`] context, used to derive default style settings.
     pub fn vertical(ui: &Ui) -> Self {
         Self::new_with_type(ui, TransitionType::VerticalMove)
     }
 
-    /// Create a new [`TransitionStyle`] with default settings mostly based on ui [style](egui::Ui::style),
-    /// but values of some fields (eg. [easing](TransitionStyle::easing)) are opinionated and may change slightly
-    /// between versions.
+    /// Creates a new [`TransitionStyle`] with default settings and [`TransitionType::HorizontalMove`].
     ///
-    /// You will mostly want to manually specify [transition type](TransitionStyle::t_type), so it's recommended
-    /// to use the [`horizontal`](TransitionStyle::horizontal) or [`vertical`](TransitionStyle::vertical)function instead.
+    /// It is generally recommended to use [`TransitionStyle::horizontal`] or [`TransitionStyle::vertical`]
+    /// to explicitly specify the transition type, improving code clarity.
+    ///
+    /// # Parameters
+    /// - `ui`: The current [`Ui`] context, used to derive default style settings.
     pub fn new(ui: &Ui) -> Self {
         Self::new_with_type(ui, TransitionType::default())
     }
 }
-/// Shows one of several possible pages with transition animation between them. The animation goes _forward_.
+
+/// Shows one of several possible pages with a forward transition animation.
+///
+/// When the `target_page` changes, a forward animation will be triggered to transition to the new page.
+/// "Forward" animation direction is determined by the provided `invert_direction` closure in [`animated_pager_with_direction`].
+/// In `animated_pager_forward`, the direction is always considered "forward".
```

Comment:
```suggestion
/// When the `target_page` changes, a "forward" animation will be triggered to transition to the new page.
```
> "Forward" animation direction is determined by the provided `invert_direction` closure in [`animated_pager_with_direction`].

This line seems irrelevant to `animated_pager_forward` imo, correct me if I'm wrong